draft-demarco-nonce-endpoint.md

```diff
@@ -68,7 +68,9 @@ The **Nonce Endpoint** MUST satisfy the following requirements:
 
 The **Nonce** MUST satisfy the following requirements:
 
-- Pure opacity to recieving Clients.
+- It MUST be opaque to receiving Clients.
+- It MUST be unguessable, by cryptographic randomness or encryption.
+- It MUST be used once and for a short time.
```

Comment:
```suggestion
- It MUST have a short validity period
```

Reply:
since both short validity and usable once are requirements I propose to slit that sentence in two distinct entries, see my proposal below

Reply:
https://github.com/peppelinux/draft-demarco-oauth-nonce-endpoint/pull/17/files#r1481757786